### PR TITLE
fix SetGridField grid field with null

### PR DESCRIPTION
**Pull request by anara123**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://github.com/demisto/etc/issues/46567)

## Description
Fixed an issue where empty grid field returned null, which caused the script to raise exception.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CommonScripts/Scripts/SetGridField/SetGridField.py [hunk 1]
```diff
@@ -130,13 +130,14 @@ def get_current_table(grid_id: str) -> pd.DataFrame:
     Returns:
         DataFrame: Existing grid data.
     """
-    current_table: Optional[List[dict]] = demisto.incidents()[0].get("CustomFields", {}).get(grid_id)
-    if current_table is None:
+    if grid_id not in demisto.incidents()[0].get("CustomFields", {}):
```
Comment (yaakovi) [inline Packs/CommonScripts/Scripts/SetGridField/SetGridField.py:133]:
```suggestion
    custom_fields = demisto.incident().get("CustomFields", {}) 
    if grid_id not in custom_fields:
```

File: Packs/CommonScripts/Scripts/SetGridField/SetGridField.py [hunk 2]
```diff
@@ -130,13 +130,14 @@ def get_current_table(grid_id: str) -> pd.DataFrame:
...
         raise ValueError(f"The following grid id was not found: {grid_id}. Please make sure you entered the correct "
                          f"incident type with the \"Machine name\" as it appears in the incident field editor in "
                          f"Settings->Advanced ->Fields (Incident). Also make sure that this value appears in the "
                          f"incident Context Data under incident - if not then please consult with support.")
 
+    current_table: Optional[List[dict]] = demisto.incidents()[0].get("CustomFields", {}).get(grid_id)
```
Comment (yaakovi) [inline Packs/CommonScripts/Scripts/SetGridField/SetGridField.py:139]:
```suggestion
    current_table: Optional[List[dict]] = custom_fields.get(grid_id)
```

File: Packs/PaloAltoNetworks_PAN_OS_EDL_Management/Integrations/PaloAltoNetworks_PAN_OS_EDL_Management/PaloAltoNetworks_PAN_OS_EDL_Management.yml [hunk 3]
```diff
@@ -28,7 +28,7 @@ configuration:
   name: document_root
   required: false
   type: 0
-description: Deprecated. Use the Palo Alto Networks PAN-OS EDL Service integration instead. This integration is still supported however, for customers with over 1000 Firewalls.
+description: Deprecated. Use the Generic Export Indicators Service integration instead. This integration is still supported however, for customers with over 1000 Firewalls.
```
Comment (yaakovi) [inline Packs/PaloAltoNetworks_PAN_OS_EDL_Management/Integrations/PaloAltoNetworks_PAN_OS_EDL_Management/PaloAltoNetworks_PAN_OS_EDL_Management.yml:31]: Is that intentional?

File: Packs/CommonScripts/ReleaseNotes/1_6_34.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### SetGridField
+- Fixed an issue where empty grid field returned null, which caused the script to raise exception.
```
Comment (yaakovi) [inline Packs/CommonScripts/ReleaseNotes/1_6_34.md:4]:
```suggestion
- Fixed an issue where an empty grid field returned null, which caused the script to raise an exception.
```

File: Packs/PaloAltoNetworks_PAN_OS_EDL_Management/Integrations/PaloAltoNetworks_PAN_OS_EDL_Management/PaloAltoNetworks_PAN_OS_EDL_Management.yml (hunk 3 above)
Comment (anara123) [reply Packs/PaloAltoNetworks_PAN_OS_EDL_Management/Integrations/PaloAltoNetworks_PAN_OS_EDL_Management/PaloAltoNetworks_PAN_OS_EDL_Management.yml:31]: This is another small fix. I will open a separate PR for it

**Comment by anara123** [comment]
The build is failing on to install `Change_Management` pack, which is not in the marketplace for the moment, because submitted few hours ago. 
```
[2022-02-09 19:15:44] - [Thread-10] - [ERROR] - Search request for pack "Change Management" with ID "Change_Management", failed.
Traceback (most recent call last):
  File "/builds/xsoar/content/Tests/Marketplace/search_and_install_packs.py", line 154, in search_pack
    response_data, status_code, _ = demisto_client.generic_request_func(client,
  File "/builds/xsoar/content/venv/local/lib/python3.9/site-packages/demisto_client/__init__.py", line 259, in generic_request_func
    return self.api_client.call_api(
  File "/builds/xsoar/content/venv/local/lib/python3.9/site-packages/demisto_client/demisto_api/api_client.py", line 327, in call_api
    return self.__call_api(resource_path, method,
  File "/builds/xsoar/content/venv/local/lib/python3.9/site-packages/demisto_client/demisto_api/api_client.py", line 158, in __call_api
    response_data = self.request(
  File "/builds/xsoar/content/venv/local/lib/python3.9/site-packages/demisto_client/demisto_api/api_client.py", line 349, in request
    return self.rest_client.GET(url,
  File "/builds/xsoar/content/venv/local/lib/python3.9/site-packages/demisto_client/demisto_api/rest.py", line 235, in GET
    return self.request("GET", url,
  File "/builds/xsoar/content/venv/local/lib/python3.9/site-packages/demisto_client/demisto_api/rest.py", line 229, in request
    raise ApiException(http_resp=r)
demisto_client.demisto_api.rest.ApiException: (400)
Reason: Bad Request
```

Regarding `PCAP Search test` which is also failed in 6.2 build:
After running the script again, it passed. It failed on timeout before.
image

**Review by yaakovi — APPROVED** [review]

**Pull request merged**